cdc/sink/mq.go

```diff
@@ -106,22 +108,37 @@ func newMqSink(
 			avroEncoder.SetValueSchemaManager(valueSchemaManager)
 			return avroEncoder
 		}
+	case codec.ProtocolCanal:
+		if !config.EnableOldValue {
+			log.Warn("mqSink View all updates as inserts if enable old value is not enabled")
+		}
+		var forceHkPk bool
+		forceHKeyToPKey, ok := opts["force-handle-key-pkey"]
+		if !ok || forceHKeyToPKey == "false" {
+			forceHkPk = false
+		} else if forceHKeyToPKey == "true" {
+			forceHkPk = true
+		}
```

Comment:
```suggestion
		if ok && forceHKeyToPKey == "true" {
			forceHkPk = true
		}
```
btw default value for bool is false